var/spack/repos/builtin/packages/lapackpp/package.py

```diff
@@ -45,9 +45,19 @@ class Lapackpp(CMakePackage):
 
     def cmake_args(self):
         spec = self.spec
+
+        blas_vendor = ''
+        cxx_flags = ' '
+        if (spec['blas'].name == 'cray-libsci'):
+            blas_vendor = '-DBLA_VENDOR=CRAY'
+        if (spec.satisfies('%cce')):
+            cxx_flags = '-DCMAKE_CXX_FLAGS=-DLAPACK_FORTRAN_ADD_'
```

Comment:
Manually specifying cmake flags like this can override other spack cxx flags. Instead, use a [flag handler](https://spack.readthedocs.io/en/latest/packaging_guide.html#compiler-flags). (You can grep other packages for `%cce` to see how this is used in practice.)